go.mod

```diff
@@ -1,6 +1,6 @@
 module helm.sh/helm/v3
 
-go 1.22.0
+go 1.22.7
```

Comment:
This changes the _minimum_ version used to build Helm from a compatibility standpoint. That's what the `go` directive.

We don't want to change the minimum version of compatibility. We want to build with the newer version.

Reply:
got it will close this PR, thanks @mattfarina